core/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -223,8 +223,13 @@ public void testTimeout() throws IOException {
                 new AliasFilter(null, Strings.EMPTY_ARRAY),
                 1.0f),
             null);
-        // the search context should inherit the default timeout
-        assertThat(contextWithDefaultTimeout.timeout(), equalTo(TimeValue.timeValueSeconds(5)));
+        try {
+            // the search context should inherit the default timeout
+            assertThat(contextWithDefaultTimeout.timeout(), equalTo(TimeValue.timeValueSeconds(5)));
+        } finally {
```

Comment:
these tests where annoyingly slow since they waited for timeouts since shards where still locked - this shaved 10 seconds off the test